### ISSUE TITLE
Deprecated errors

### ISSUE THREAD

**Issue by IvanovAlmeida** [dereuromark/CakePHP-DatabaseLog#27]
I'm getting depreciation errors.

CakePHP 3.6.x

``` 
Deprecated: DatabaseLog\Model\Table\DatabaseLogsTable::displayField() is deprecated. Use setDisplayField()/getDisplayField() instead. - 
C:\xampp\htdocs\pos-certificados\vendor\dereuromark\cakephp-databaselog\src\Model\Table\DatabaseLogsTable.php, line: 46
```
and
``` 
Deprecated: DatabaseLog\Model\Table\DatabaseLogsTable::connection() is deprecated. Use setConnection()/getConnection() instead. - 
C:\xampp\htdocs\pos-certificados\vendor\dereuromark\cakephp-databaselog\src\Model\Table\LazyTableTrait.php, line: 56
```

**Comment by dereuromark** [comment]
Please disable them in your app config as documented.
Or make a pr if this can be fixed for 3.5+

**Comment by dereuromark** [comment]
Fixed in https://github.com/dereuromark/CakePHP-DatabaseLog/commit/1e408a96dadda6d9a1a7c45f229b7274d43b38ec
